src/offline/cable_surface_types.F90

```diff
@@ -0,0 +1,37 @@
+!#define UM_CBL YES
```

Comment:
Why does this line say YES when this file is used within a "ndef" condition?

Reply:
OMG  - I just answered this and now I can't find where.

My recollection from many years ago when I was playing with FPP/CPP directives is that you can't leave a #definition hanging, i.e. it has to be defined as something or it is interpreted as a syntax error